test/francis_test.exs

```diff
@@ -1,5 +1,6 @@
 defmodule FrancisTest do
   use ExUnit.Case, async: true
+  @moduletag :tmp_dir
```

Comment:
It seems this is only needed on the last describe block, maybe change it to a describetag there?